src/configuration.test.ts

```diff
@@ -0,0 +1,91 @@
+import { describe } from 'riteway'
+import { camelCaseToScreamingSnakeCase, createEnvToConfigurationKeyMap, mergeConfigs } from './configuration'
+
+describe('camelCaseToScreamingSnakeCase()', async (assert) => {
+  assert({
+    given: 'no arguments',
+    should: 'return an empty string',
+    actual: camelCaseToScreamingSnakeCase(),
+    expected: '',
+  })
+
+  assert({
+    given: 'empty string',
+    should: 'return an empty string',
+    actual: camelCaseToScreamingSnakeCase(''),
+    expected: '',
+  })
+
+  assert({
+    given: 'lowercase string',
+    should: 'return an UPPERCASE string',
+    actual: camelCaseToScreamingSnakeCase('test'),
+    expected: 'TEST',
+  })
+
+  assert({
+    given: 'camelCase string',
+    should: 'return SCREAMING_SNAKE_CASE string',
+    actual: camelCaseToScreamingSnakeCase('testCaseString'),
+    expected: 'TEST_CASE_STRING',
+  })
+})
+
+describe('createEnvToConfigurationKeyMap()', async (assert) => {
+  assert({
+    given: 'empty array',
+    should: 'return empty object',
+    actual: createEnvToConfigurationKeyMap([]),
+    expected: {},
+  })
+
+  {
+    const defaultConfiguration = ['frostApiUrl', 'testUrl']
+
+    assert({
+      given: 'array with cammelCase values',
+      should: 'return object with ScreamingSnakeCase keys and equal cammelCase values',
+      actual: createEnvToConfigurationKeyMap(defaultConfiguration),
+      expected: {
+        FROST_API_URL: 'frostApiUrl',
+        TEST_URL: 'testUrl',
+      },
+    })
+  }
+})
+
+const defaultConfig = mergeConfigs()
+
+describe('src/Configuration', async (assert) => {
+  assert({
+    given: 'no arguments',
+    should: 'return the default config',
+    actual: mergeConfigs(),
+    expected: defaultConfig,
```

Comment:
Since `const defaultConfig = mergeConfigs()`, this is comparing `mergeConfigs()` against `mergeConfigs()`, which will always be true no matter what.

Let's export / import the default config and test against that.